kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -39,6 +39,7 @@
       :channelId="channelId"
       :available="content.available"
       :extraFields="content.extra_fields"
+      :hasPrimaryButton="!showNextBtn"
```

Comment:
ShowNextBtn returns an object so this does not work

![localhost_8000_learn_](https://user-images.githubusercontent.com/7193975/32971409-f9033c48-cba1-11e7-9b3a-9be25ba32fe3.png)

![image](https://user-images.githubusercontent.com/7193975/32971423-0866b9b2-cba2-11e7-91d4-b6916c5b3675.png)

Reply:
Or wait, does the screenshot show intentional behavior @indirectlylit ?

Reply:
no, that's wrong. Only 'go to next item' should be primary.

Reply:
I don't understand how this is an object...

```javascript
      showNextBtn() {
        return this.progress >= 1 && this.content && this.nextContentLink;
      },
```